pkgs/development/python-modules/pytest-docker/default.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  wheel,
+  attrs,
+  pytest,
+  docker-compose,
+  mypy,
+  pytest-mypy,
+  pycodestyle,
+  pytest-pylint,
+  requests,
+  types-requests,
+  types-setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-docker";
+  version = "3.1.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-I3FSSASnUqqnZsebnu6OY0U0r924JZfztXPafF1v+18=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    attrs
+    pytest
+  ];
```

```suggestion
  buildInputs = [ pytest ];

  dependencies = [ attrs ];
```
To make it easily overridable, `pytest` should be added to `buildInputs`.
See other pytest plugins.